manual/tmva/index.md

```diff
@@ -26,29 +31,77 @@ The package includes:
 
 {% include tutorials name="TMVA" url="tmva" %}
 
-> **Topical manuals**
->
-> For TMVA, topical manuals are available at [Topical Manuals - TMVA]({{ '/topical/#tmva' | relative_url }}).<br>
-> They contain in-depth information about TMVA.
+### Ingesting a dataset for training in TMVA
 
-> **TMVA in the ROOT forum**
->
-> Discuss TMVA in the [ROOT forum](https://root-forum.cern.ch/c/tmva/22){:target="_blank"}.
+A ROOT dataset can be easily ingested for training using a {% include ref
+class="DataLoader" namespace="TMVA" %}. A common usecase in HEP is to have the
+information of physics events split in signal and background, as in the example
+below:
 
-## Using TMVA
-
-The ROOT tutorials for TMVA, available in `$ROOTSYS/tutorials/tmva`, provide example jobs for the training phase and the application of the training results in a classification or regression analysis using the TMVA Reader.
+_**Example**_
 
+{% highlight C++ %}
+void load_data(){
+   // open file and retrieve trees
+   auto inputfile = TFile::Open("http://root.cern/files/tmva_class_example.root");
+   auto signaltree = inputfile->Get<TTree>("TreeS");
+   auto backgroundtree = inputfile->Get<TTree>("TreeB");
+
+   // load trees into TMVA
+   TMVA::DataLoader loader{"mydataloader"};
+   loader.AddSignalTree(signaltree);
+   loader.AddBackgroundTree(backgroundtree);
+}
+{% endhighlight %}
+
+The loaded data can then be passed to the {% include ref class="Factory"
+namespace="TMVA" %} class for training, as shown in
+[this tutorial](https://root.cern/doc/master/tmva003__RReader_8C.html){:target="_blank"}.
```

Comment:
I think this is not the right tutorial. The Reader is for inference. 
I think the tutorial should be `TMVAClassification.C`